teamB/kimSungHoon/services/userService.js

```diff
@@ -0,0 +1,42 @@
+const userDao = require('../models/userDao');
+
+const signUp = async (name, email, password, profile_image) => {
+
+    const pwValidation = new RegExp(
```

Comment:
문자열을 변수에 할당할 때엔 변수를 대문자로 작성하는 컨벤션이 있습니다.

```suggestion
    const PASSWORD_REGEX = new RegExp(
```

Reply:
수정 완료했습니다.